frontend/src/scenes/insights/views/BoldNumber/BoldNumber.tsx

```diff
@@ -0,0 +1,113 @@
+import { formatAggregationAxisValue } from 'scenes/insights/aggregationAxisFormat'
+import clsx from 'clsx'
+import { ensureTooltipElement } from '../LineGraph/LineGraph'
+import { groupsModel } from '~/models/groupsModel'
+import { toLocalFilters } from 'scenes/insights/filters/ActionFilter/entityFilterLogic'
+import { InsightTooltip } from 'scenes/insights/InsightTooltip/InsightTooltip'
+import { personsModalLogic } from 'scenes/trends/personsModalLogic'
+
+/** The effect of the value's padding is reduced by the offset. */
+const BOLD_NUMBER_TOOLTIP_OFFSET_PX = -16
+
+function useBoldNumberTooltip({
+    showPersonsModal,
+    isTooltipShown,
+}: {
+    showPersonsModal: boolean
+    isTooltipShown: boolean
+}): React.RefObject<HTMLDivElement> {
+    const { filters, insight } = useValues(insightLogic)
+    const { aggregationLabel } = useValues(groupsModel)
+
+    const divRef = useRef<HTMLDivElement>(null)
+
+    useEffect(() => {
+        const divRect = divRef.current?.getBoundingClientRect()
+        const tooltipEl = ensureTooltipElement()
+        tooltipEl.style.opacity = isTooltipShown ? '1' : '0'
+
+        const seriesResult = insight.result[0]
+
+        ReactDOM.render(
+            <InsightTooltip
+                renderCount={(value: number) => (
+                    <>{formatAggregationAxisValue(filters.aggregation_axis_format, value)}</>
+                )}
+                seriesData={[
+                    {
+                        dataIndex: 1,
+                        datasetIndex: 1,
+                        id: 1,
+                        label: seriesResult.label,
+                        count: seriesResult.aggregated_value,
+                    },
+                ]}
+                showHeader={false}
+                hideColorCol
+                hideInspectActorsSection={!showPersonsModal}
+                groupTypeLabel={aggregationLabel(toLocalFilters(filters)[0].math_group_type_index).plural}
+            />,
+            tooltipEl,
+            () => {
+                const tooltipRect = tooltipEl.getBoundingClientRect()
+                if (divRect) {
+                    tooltipEl.style.top = `${
+                        window.scrollY + divRect.top - tooltipRect.height - BOLD_NUMBER_TOOLTIP_OFFSET_PX
+                    }px`
+                    tooltipEl.style.left = `${divRect.left + divRect.width / 2 - tooltipRect.width / 2}px`
+                }
+            }
+        )
+    }, [isTooltipShown])
+
+    return divRef
+}
+
+export function BoldNumber({ showPersonsModal = true }: ChartParams): JSX.Element {
+    const { insight, filters } = useValues(insightLogic)
+    const { loadPeople } = useActions(personsModalLogic)
+
+    const [isTooltipShown, setIsTooltipShown] = useState(false)
+    const valueRef = useBoldNumberTooltip({ showPersonsModal, isTooltipShown })
+
+    const resultSeries = insight.result[0] as TrendResult
+
+    return (
+        <div className="BoldNumber">
```

Comment:
nit: some of the styling inside the `BoldNumber` scss could be utility classes

Reply:
That is a good point and I was thinking about which way to go. The issue is, I can't use breakpoints and I can't use `letter-spacing` with utility classes, so I would have to do `.BoldNumber__value` anyway, at which point I thought  "well, in this case it will just be more readable with the classic BEM rather than mixing it with Tailwind". I would love to use just utility classes here, but our system isn't expressive enough yet.

Reply:
Sounds about right! Until we actually have tailwind (and even in advanced cases anyways) I think BEM style css makes sense for these cases